frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.tolerations.scenario.ts

```diff
@@ -0,0 +1,52 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { createResource, deleteResource, click } from '@console/shared/src/test-utils/utils';
+import * as editTolerationsView from '../views/editTolerationsView';
+import * as virtualMachineView from '../views/virtualMachine.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { getVMManifest } from './utils/mocks';
+import { getRandStr } from './utils/utils';
+
+describe('KubeVirt VM detail - edit Tolerations', () => {
+  const testVM = getVMManifest('Container', testName, `tolerations-vm-${getRandStr(5)}`);
+  const vm = new VirtualMachine(testVM.metadata);
+
+  beforeAll(async () => {
+    createResource(testVM);
+  });
+
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it(
```

Comment:
Similarly to Node Selector tests, we should probably test node matching works as expected